www/js/AvailableSite.js

```diff
@@ -1,41 +1,179 @@
+            tx.executeSql('select siteId, siteName, circle, state from SITE', [], function(tx, rs){
+                permission_list=rs.rows;    
+            });
+        }, function(error){
+            alert("Transaction Error: "+error.message);
+        }, function(){
+                //alert("11:"+permission_list.length);
+                if(permission_list.length>0){
+                var insertlist=" ";
+                for(var i=0; i<permission_list.length; i++){
+                        insertlist+="<hr><li class='survey_item'><h5>Site: "+permission_list.item(i).siteName+"</h5><h5>State: "+permission_list.item(i).state+"</h5><h5>Circle#: "+permission_list.item(i).circle+"</h5></li> "
                 }
-                    
-         }else{
-                window.alert("Delete unsuccessfully");
-        
-         }   
-   }
-  )
-});
\ No newline at end of file
+                $("#permissionList").html(insertlist);
+                //"<li class='survey_item1'><h5>Site: "++"</h5><h5>Site Id: "++"</h5></li> "
+                }else{
+                        $("#permissionList").html("<li class='survey_item'><h5>You Do Not Have Any Site Permission</h5><h5>Please contact Site Manager for detail</h5></li>");
+                }
+        });
+};
+
+
+$(document).ready(function(){
+    var $newsite = $(".newsite");
+    $newsite.click(function() {
+                var password=window.prompt("Please enter passward for site selected");
+                var siteID = $("#site option:selected").val();
+                //validate site password
+                $.ajax({
+		url: DOMAIN + "/api/sites.php",
+		type : "POST",
+		crossDomain: true,
+		dataType: 'json',
+		data: JSON.stringify({
+			"action": "checkSitePassword",
+			"siteID": siteID,
+			"sitePasswordCheck": password
+		}),
+		success: function(passwordCheckResult){
+			//navigator.notification.alert(passwordCheckResult.validSitePassword);
+			//If site password is correct, submit survey
+			if(passwordCheckResult.validSitePassword == 1){
+				//navigator.notification.alert("Site password correct");
+				pushSitetoDB(siteID);
+			}
+			else{
+				navigator.notification.alert("Site password is incorrect.");
+			}
+		},
+		error : function(){
+			navigator.notification.alert("Unexpected error checking site password.");
+		}
+	});
+    });
+
+    $("#reset").click(function(){
+        db.transaction(function(tx){
+        tx.executeSql('DROP TABLE IF EXISTS SITE' );
+        tx.executeSql('CREATE TABLE IF NOT EXISTS SITE (siteId, siteName, userId, circle, state)' );
+        }, function(error){
+            alert("Transaction Error: "+error.message);
+        }, function() {
+            alert("successfully Cleared all the information in Table SITE");
+            retriveSitePermission();
+
+    });
+    });
+});
+//CREATE TABLE IF NOT EXISTS SITE (siteId, siteName, userId, circle, state)
+function pushSitetoDB(siteID){
+        //get # of circle of specific site
```

Comment:
The name here is a bit misleading. It seems like you are trying to get the number of circles pushed in, so the name should actually be 'getCircleCount'